### DISCUSSION TITLE
Can APK files not be packaged over 2GB?

### DISCUSSION THREAD

**Discussion by Jason0247** [airsdk/Adobe-Runtime-Support#2030] [Q&A]
I tried to package more than 2gb of APK files, but still can not install and release, is this the AIR SDK limit? Is there any way to set this limit? Thank you!

**Comment by ajwfrost** [comment]
I'm not sure whether there's a limit to the tools but you wouldn't be able to publish a file like that:
https://medium.com/unpacking-react-native/google-play-store-app-size-f7a1b9fb58d0
https://support.google.com/googleplay/android-developer/answer/9859152?visit_id=637946734793329821-1299132660&rd=1#apk&zippy=%2Cmaximum-size-limit

It's possible we're using the original 32-bit zip format (APKs are just zip files after all) although that should mean the limit is for 4GB (of uncompressed data). So if you're not planning to publish your APK on the app store, and definitely want to do this, can you please enable the debug logging (in the AIR SDK "lib" folder, edit "adt.cfg" to remove the comment hash character from "DebugOut=true") and then try again, and send us the log file (from c:\users\username\adt.log on Windows, or ~/adt.log on mac).

thanks

**Comment by Jason0247** [comment]
The latest version of the air SDK I just used can pack APK files of more than 2G, which may be the old version I used before.
I'll try to see if the packaged APK can be installed on my mobile device later, and I'll feed back the results later!

**Comment by Jason0247** [comment]
[adt.log](https://github.com/airsdk/Adobe-Runtime-Support/files/9217688/adt.log)
Tested, APK can be packaged, but cannot be installed on mobile devices.
Prompt "there is a problem parsing the package" during installation @ajwfrost
